encoding/kzg/pointsIO.go

```diff
@@ -54,7 +54,7 @@ func ReadG1Point(n uint64, g *KzgConfig) (bn254.G1Affine, error) {
 
 // Read the n-th G2 point from SRS.
 func ReadG2Point(n uint64, g *KzgConfig) (bn254.G2Affine, error) {
-	if n > g.SRSOrder {
+	if n >= g.SRSOrder {
```

Comment:
If the SRS with order n is up to [s^n], then there are n+1 elements for order n (since it's starting from [s^0]). So n can be equal to g.SRSOrder, right?

Reply:
In our convention, SRS order is the number of point, starting at point 0.